### PR TITLE
fix(import-notation): stringify without duplicates

### PR DESCRIPTION
**Pull request by Yeti-or**

### CODE REVIEW

File: packages/import-notation/index.js [hunk 1]
```diff
@@ -96,7 +96,10 @@ function stringify(cells) {
         cell.block && (acc.b = cell.block);
         cell.elem && (acc.e = cell.elem);
         cell.mod && (acc.m[cell.mod.name] || (acc.m[cell.mod.name] = []))
-            && cell.mod.val && typeof cell.mod.val !== 'boolean' && acc.m[cell.mod.name].push(cell.mod.val);
+            && cell.mod.val
+                && typeof cell.mod.val !== 'boolean'
+                && !~acc.m[cell.mod.name].indexOf(cell.mod.val)
```
Comment (Vittly) [inline packages/import-notation/index.js]: Hacker!

File: packages/import-notation/index.js [hunk 2]
```diff
@@ -96,7 +96,10 @@ function stringify(cells) {
         cell.block && (acc.b = cell.block);
         cell.elem && (acc.e = cell.elem);
         cell.mod && (acc.m[cell.mod.name] || (acc.m[cell.mod.name] = []))
-            && cell.mod.val && typeof cell.mod.val !== 'boolean' && acc.m[cell.mod.name].push(cell.mod.val);
+            && cell.mod.val
+                && typeof cell.mod.val !== 'boolean'
```
Comment (Vittly) [inline packages/import-notation/index.js]: extra offset? Why not

```js
  && cell.mod.val
  && typeof cell.mod.val !== 'boolean'
  // etc
```

File: packages/import-notation/test/stringify.test.js [hunk 3]
```diff
@@ -66,6 +66,23 @@ describe('block', () => {
             { block : 'popup', mod : { name : 'autoclosable', val : 'yes' } }
         ])).to.equal('b:popup m:theme=normal|action m:autoclosable=yes');
     });
+
+    it('should not duplicate entities', () => {
+        expect(s([
+            { block : 'popup' },
+            { block : 'popup' },
+            { block : 'popup', mod : { name : 'theme' } },
```
Comment (Vittly) [inline packages/import-notation/test/stringify.test.js:74]: Why don't we check `elem` dups?

Comment (Yeti-or) [reply packages/import-notation/test/stringify.test.js:74]: cause actually they are same
but I could do it for your pleasure

File: packages/import-notation/index.js (hunk 1 above)
Comment (Yeti-or) [reply packages/import-notation/index.js]: whole module written like this =)

Comment (Yeti-or) [reply packages/import-notation/index.js]: whole module written like this =)

File: packages/import-notation/test/stringify.test.js (hunk 3 above)
Comment (Vittly) [reply packages/import-notation/test/stringify.test.js:74]: nice!

File: packages/import-notation/index.js (hunk 1 above)
Comment (qfox) [reply packages/import-notation/index.js]: whole module written like this =)

Comment (Vittly) [reply packages/import-notation/index.js]: 😆

**Review by qfox — APPROVED** [review]

**Pull request merged**